packages/scoped-elements/test-web/lit-element.test.js

```diff
@@ -134,4 +136,13 @@ describe('lit-element ScopedElementsMixin', () => {
     expect(stepB.constructor).to.equal(MyButton2);
     expect(stepB.color).to.equal('blue');
   });
+
+  it('creates unique registries when one lit element inherits another lit element', async () => {
```

Comment:
If you've changed `MyButton3` to use HTMLElement instead of LitElement, can we move this test to `html-element.test.js`?

Reply:
Should I keep `FeatureC` in the `lit` directory and then import it in the `test` file from `'./components/vanilla/index.js'`? Or should I move it to `vanilla` directory and update its rendering part to use `shadowRoot` directly?